--- configure.ac.original	2018-01-02 04:35:40.934327216 +0900
+++ configure.ac	2018-01-02 04:37:33.747164519 +0900
@@ -555,16 +555,11 @@ AC_CHECK_LIB(socket, setsockopt)
 if test "$have_cpp" = "yes" ; then
 # mingw toolchain used to build "Thrift Compiler for Windows"
 # does not support libcrypto, so we just check if we building the cpp library
-AC_CHECK_LIB(crypto,
-    BN_init,
-    [AC_CHECK_LIB(ssl,
+    AC_CHECK_LIB(ssl,
         SSL_ctrl,
-        [LIBS="-lssl -lcrypto $LIBS"],
-        [AC_MSG_ERROR(["Error: libssl required"])],
-        -lcrypto
-    )],
-    [AC_MSG_ERROR(["Error: libcrypto required."])]
-)
+        [LIBS="-lssl $LIBS"],
+        [AC_MSG_ERROR(["Error: libssl required"])]
+    )
 fi
 
 AC_TYPE_INT16_T
