--- volk/lib/CMakeLists.txt.original	2017-12-29 21:04:53.737124882 +0900
+++ volk/lib/CMakeLists.txt	2017-12-29 21:07:04.966182993 +0900
@@ -429,7 +429,7 @@ if(${CMAKE_VERSION} VERSION_GREATER "2.8
       message(STATUS "---- Adding ASM files") # we always use ATT syntax
       message(STATUS "-- Detected neon architecture; enabling ASM")
       # setup architecture specific assembler flags
-      set(ARCH_ASM_FLAGS "-mfpu=neon -g")
+      set(ARCH_ASM_FLAGS "-march=armv7-a -mtune=cortex-a53 -mthumb -mfloat-abi=hard -mfpu=neon -g --param ggc-min-expand=1 --param ggc-min-heapsize=32768")
       # then add the files
       include_directories(${PROJECT_SOURCE_DIR}/kernels/volk/asm/neon)
       file(GLOB asm_files ${PROJECT_SOURCE_DIR}/kernels/volk/asm/neon/*.s)
